tools/misc/calc_timeprop_spectrum

```diff
@@ -15,17 +15,24 @@ import argparse
 import numpy as np
 from scipy import constants
 
-USAGE = """Reads output from TD calculation (after kick) and produces
-absorption spectra. Needs mux.dat, muy.dat and muz.dat files in working directory."""
+USAGE = """%(prog)s -d xx
```

Comment:
That's not a very useful "usage" message. Use the one here, which one would get, if argumentparser had generated the message automatically (when `usage=USAGE` is used)